src/fracas/hover-provider.ts

```diff
@@ -17,15 +17,16 @@ export class FracasHoverProvider implements vscode.HoverProvider {
     ): Promise<vscode.Hover | null> {
         // search for comments preceding the definition for the symbol at the cursor
         const definitions = await findDefinition(document, position, token);
-        const comments = await mapAsync(definitions, async (definition) => {
+        const comments_raw = await mapAsync(definitions, async (definition) => {
             return await findComment(definition.location.uri, definition.location.range.end);
         });
+        const comments = comments_raw.join("\n").trim();
 
         // return a hover containing the comments, if any.
         const positionAsRange = new vscode.Range(position, position);
         if (comments.length > 0) {
             const resolvedSymbol = resolveSymbol(document, position);
-            return new vscode.Hover(comments.join("\n"), resolvedSymbol?.range || positionAsRange);
+            return new vscode.Hover(comments, resolvedSymbol?.range || positionAsRange);
```

Comment:
This was being called with "empty" comments consisting of 2 empty lines. This change consolidates them and measures the emptiness correctly.